subsys/logging/Kconfig

```diff
@@ -362,6 +362,13 @@ config LOG_BACKEND_NATIVE_POSIX
 	help
 	  Enable backend in native_posix
 
+config LOG_BACKEND_QEMU_X86_64
+	bool "Enable QEMU x86_64 backend"
+	depends on BOARD_QEMU_X86_64
+	default y
```

Comment:
Because it's the default logger for BOARD_QEMU_X86_64. It will only be enabled if that board is selected

Reply:
Shouldn't we, then, just select this in the board definition?

Reply:
Actually it's a good point. I assume there's no default UART backend on x86_64, but maybe this should be:
default y if BOARD_QEMU_X86_64

Reply:
I believe anything that is board specific should be at least _selected_ at the board definition, rather than using default y. What is there are multiple backends for _64 in the future? Would we need to set this one to =n?

Reply:
removed `default y` and moved setting it to  `qemu_x86_64_defconfig` file.